definitions/ext-network_dhcp_service/giga-dhcpv4-dashboard.json

```diff
@@ -0,0 +1,195 @@
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Gigamon select count(*) WHERE app_name = 'dhcp' AND instrumentation.provider = 'Gigamon' FACET dhcp_message_type "
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Request Count by Domain",
+          "layout": {
+            "column": 1,
+            "row": 5,
+            "width": 6,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Gigamon Select uniqueCount(dhcp_host_name) AS 'DHCP Clients (count)' WHERE app_name = 'dhcp' AND instrumentation.provider = 'Gigamon' AND dhcp_message_type = 'DHCPREQUEST' FACET dhcp_domain_name "
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Lease Duration by DHCP Domain",
+          "layout": {
+            "column": 7,
+            "row": 5,
+            "width": 6,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Gigamon Select max(numeric(dhcp_ip_lease_time))/60 AS 'Longest Lease (minutes)'  WHERE app_name = 'dhcp' AND instrumentation.provider = 'Gigamon' AND dhcp_message_type = 'DHCPREQUEST' FACET dhcp_domain_name"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "DHCP Transaction Details (forwarded by router)",
+          "layout": {
+            "column": 1,
+            "row": 8,
+            "width": 6,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Gigamon Select timestamp, dhcp_message_type, src_ip, src_mac, src_packets, src_port, src_bytes, dst_ip, dst_mac, dst_packets, dst_port, dst_bytes, end_reason, end_time, generator, id, protocol, seq_num, start_time, ts, udp_client_port, udp_server_port, vendor, version WHERE app_name = 'dhcp' AND instrumentation.provider = 'Gigamon' AND dhcp_hops != '0' SINCE 1 DAY AGO"
```

Comment:
Why fixed SINCE and not letting the user use his defined timerange?

Reply:
As above